docs/getting-started-with-openshift.md

```diff
@@ -0,0 +1,147 @@
+
+Docker provides a number of ways in which to manage your docker image files, and include the following commands:
+
+The typical workflow with docker will be to search/ choose which docker images you wish to use. These images are hosted on DockerHub and docker allows you to search for images, and either pull/ download those images to your local machine or specify those images in DockerFile's to be used to construct containers. Docker images on your local machine can then be searched for via the docker CLI. Useful commands include:
+
+* `docker search <name> # searches docker registry for docker image`
+* `docker search <search-string> | grep <grep-string> # narrows down search`
+* `docker pull <image name> # pulls image from docker registry and downloads it`
+* `docker images # shows all local docker images`
+* `docker images | grep <grep-string> # narrows down list with grep`
+* `docker images -a # show all images on this machine`
+* `docker rmi <image-name> # remove specified image from this machine`
+
+In order to push your own images to the docker registry, you should tag then, and then push them. You can also run an image directly from the registry:
+
+* `docker tag <image> username/repository:tag # Tag <image> for upload to registry`
+* `docker push username/repository:tag # Upload tagged image to registry`
+* `docker run username/repository:tag # Run image from a registry`
+
+### Persistence of container data ###
+
+Docker containers are designed to be spun up and torn down quickly. When a container is torn down all data in that container goes with it. To persist data from your docker container, involves creating a folder on the docker host system and mounting that folder within the docker container, a little like a network drive. This host folder may be mounted within several containers, with all containers reading/ writing to the same data source. 
+
+First set up a volume on your docker host system:
+* `mkdir /var/dbfiles # make host folder`
+* `chmod o+rwx /var/dbfiles # open up permissions for folder`**
+* `chcon -t svirt_sandbox_file_t /var/dbfiles # change SELinux security context of the new /var/dbfiles directory`
+
+Once the volume has been set up on your host system, it can be mapped to the container by adding `-v <host-system-volume>:<folder-on-container>` when you run your container with `docker run`. An example of this with a mysql container might be something like: 
+* `docker run -d --name mysql -e MYSQL_DATBASE=items -e MYSQL_USER=user1 -p 30306:3306 -v /var/local/mysql:var/lib/mysql mysql`
+
+## Coordinating multi-container applications ##
+
+
+
+## Orchestrating containers with Kubernetes ##
+
+
+
+## Enterprise featues of OpenShift ##
```

Comment:
May ultimately leave out enterprise-y stuff out of the doc if we're pitching it at as a doc for community-type devs to get up and running with OpenShift

For the moment will leave it in, and we can review what the finished set of docker/ openshift docs will look like